src/Raven.Studio/typescript/widgets/virtualGrid/checkedColumn.ts

```diff
@@ -0,0 +1,26 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Checked cell template used for toggling selection of a row in the virtual grid.
+ */
+class checkedColumn implements virtualColumn {
+    public readonly cellClass = "checked-cell";
```

Comment:
Yep, understood. I suggest we move to upper case classes at the time we migrate the Studio to Angular/React/Aurelia post-4.0.